src/apps/lwaftr/lwaftr.lua

```diff
@@ -238,12 +238,29 @@ local function init_transmit_icmpv4_reply (rate_limiting)
    end
 end
 
+function select_instance(conf, device)
+   -- Merges t1 into t2 by mutating t1 with t2's values. This is quicker than
+   -- producing a new table with the combination of t1 & t2 like I'd have liked.
+   local function table_merge(t1, t2)
+      for k,v in pairs(t2) do t1[k] = v end
+   end
+
+   instance = assert(conf.softwire_config.instance[device])
+
+   table_merge(conf.softwire_config.external_interface,
```

Comment:
I am not sure that we should be mutating the incoming configuration, dunno.  We should copy it if we need to mutate it IMO.